apps/omg_watcher/test/omg_watcher/integration/block_getter_test.exs

```diff
@@ -48,83 +48,6 @@ defmodule OMG.Watcher.Integration.BlockGetterTest do
 
   @moduletag timeout: 100_000
 
-  @tag timeout: 200_000
-  @tag fixtures: [:in_beam_watcher, :mix_based_child_chain, :alice, :bob, :alice_deposits, :token]
-  test "get the blocks from child chain after sending a transaction and start exit",
```

Comment:
Covered by https://github.com/omisego/elixir-omg/blob/master/priv/cabbage/apps/itest/test/itest/standard_exit_test.exs

Reply:
meh missing the utxos checking part, adding to cabbage

Reply:
The whole PR is for removing this code block